components/CounterAlert/index.js

```diff
@@ -4,10 +4,10 @@ import { pluralizeAndCount, pluralize } from 'util/pluralize';
 import EyeIcon from 'components/EyeIcon';
 
 export default function CounterAlert({ status, count, ...props }) {
-  let label = statusLabel[status].toLowerCase();
-  label =
+  status = status ? status : 'null';
```

Comment:
De repente sirve:
`export default function CounterAlert({ status = 'null', count, ...props }) {` (solo que status tiene que ser `undefined` para que pase al valor por defecto.

ya de por sí usar 'null' como un valor propio de nuestra lógica me hace sentir the chills jaja pero no sé que tanto cueste hacer ese cambio, solo usar el default param @LuighiV, gracias!

Reply:
Si, bueno en realidad para el caso de status, habían dos opciones el null que era cuando no tenía evento registrado y el 'null' que es cuando el evento registrado en realidad no tiene una alerta, que sería como el 'null' state. Si también me quedé pensando en qué nombre ponerle al estado sin alerta, pense también en 'none', pero al final me quedé con 'null'. Quizás tienes pensado un mejor nombre para este estado, sin alerta.

Reply:
Hmm pensando rápido diría que una opción para sin estado sería 'stateless' o 'eventless'.

Reply:
dale, creo que lo puedo cambiar a `stateless`

Reply:
changed in [7692374](https://github.com/openpolitica/proclama-preliminar/pull/100/commits/7692374df19b880b01f4b22ee8238d288984e7ed)

Reply:
cool!